tools/dotnet-deb-tool/ArgumentEscaper.cs

```diff
@@ -0,0 +1,206 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dotnet.Deb.Tool
+{
+    public static class ArgumentEscaper
```

Comment:
If I were to do it, I would just shared-source-compile it from the build_project.  That way you don't have an external dependency to deal with.  You only have an internal dependency on the code living in the build_project, which could easily be duped at a later date.